spec/parsers/bulkrax/bagit_parser_spec.rb

```diff
@@ -274,117 +274,12 @@ module Bulkrax
     end
 
     context 'when exporting a bagit file' do
+      # Use OpenStructs to simulate the behavior of ActiveFedora::SolrHit instances.
       subject { described_class.new(exporter) }
       let(:exporter) { FactoryBot.create(:bulkrax_exporter_worktype_bagit) }
       let(:work_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
       let(:collection_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
       let(:file_set_ids_solr) { [OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9)), OpenStruct.new(id: SecureRandom.alphanumeric(9))] }
-      let(:parent_record_1) { build(:work) }
-      let(:parent_record_2) { build(:another_work) }
-
-      before do
-        allow(parent_record_1).to receive(:file_set_ids).and_return([file_set_ids_solr.pluck(:id).first])
-        allow(parent_record_1).to receive(:member_of_collection_ids).and_return([collection_ids_solr.first.id])
-        allow(parent_record_2).to receive(:file_set_ids).and_return([])
-        allow(ActiveFedora::Base).to receive(:find).with(work_ids_solr.first.id).and_return(parent_record_1)
-        allow(ActiveFedora::Base).to receive(:find).with(work_ids_solr.last.id).and_return(parent_record_2)
-      end
-
-      describe '#find_child_file_sets' do
-        before do
-          subject.instance_variable_set(:@file_set_ids, [])
-        end
-
-        it 'returns the ids when child file sets are present' do
-          subject.find_child_file_sets(work_ids_solr.pluck(:id))
-          expect(subject.instance_variable_get(:@file_set_ids)).to eq([file_set_ids_solr.pluck(:id).first])
-        end
-      end
-
-      describe '#create_new_entries' do
```

Comment:
this method is now inherited from the csv parser so I moved it to the csv parser spec